### PR TITLE
feat: add updateOperator

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

**Comment by cazala** [comment]
looks like the linter is complaining ☝️

**Comment by NicoSantangelo** [comment]
Two things:
- To add a new validation to the sanity check, you should go to https://github.com/decentraland/marketplace/blob/master/sanity/doctors/ParcelDoctor.js#L45. The `ParcelDiagnosis` will take care of the fixing part in this case, so no code needed there, if you want, you could add an `update` removing `update_operator` [here](https://github.com/decentraland/marketplace/blob/master/sanity/doctors/ParcelDoctor.js#L76), but I think that if the blockchain events are re-fetched it should work fine.
- The tests are failing just because we need to add `update_operator` to the expected object

File: migrations/1538238012345_parcels-add-update-operator.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Parcel } from '../src/Parcel'
+import { Estate } from '../src/Estate'
```
Comment (cazala) [inline migrations/1538238012345_parcels-add-update-operator.js]: The above two come from `../src/Asset` (as of #492)

**Comment by cazala** [comment]
@eordano looking good but need rebase

**Pull request merged**